packages/scaffolding-cli/src/domain/config/questions.ts

```diff
@@ -130,25 +130,25 @@ export function cliAdvancedQuestions(): Array<PromptQuestion> {
         {
             "type": "text",
             "name": "business_company",
-            "message": "Please provide the company name (will be used namespacing conventions)",
+            "message": "Please provide the company name (will use namespacing conventions)",
             "initial": "amido"
         },
         {
             "type": "text",
             "name": "business_project",
-            "message": "Please provide the project name (will be used namespacing conventions)",
+            "message": "Please provide the project name (will use namespacing conventions)",
             "initial": "stacks"
         },
         {
             "type": "text",
             "name": "business_component",
-            "message": "Please provide the component name (will be used namespacing conventions)",
+            "message": "Please provide the component name (will use namespacing conventions)",
             "initial": "cycle2"
         },
         {
             "type": "text",
             "name": "terraform_backend_storage",
-            "message": "Terraform backend state storage (Blob name)?",
+            "message": "Terraform backend state storage (blob name)?",
```

Comment:
Should we do this as optional if sleecting vsts?

Reply:
Do you mean changing the flow? Or just giving an option for 'Not required' (or similar).